process-controller/src/main/java/org/jboss/as/process/logging/ProcessLogger.java

```diff
@@ -820,4 +820,12 @@
             "using java.lang.Process.destroyForcibly()")
     void destroyingProcess(String process, long timeout);
 
+    /**
+     * Logs an error message indicating the process failed to start, along with exception associated to the failure.
+     *
+     * @param processName the process name.
+     */
+    @LogMessage(level = ERROR)
+    @Message(id = 9, value = "Failed to start process '%s': %s")
+    void failedToStartProcess(String processName, Throwable t);
```

Comment:
This won't log a stack trace. Please change the existing method to

```
    @LogMessage(level = ERROR)
    @Message(id = 9, value = "Failed to start process '%s'")
    void failedToStartProcess(@Cause Throwable cause, String processName);
```

Reply:
@bstansberry fyi this is now fixed.